docs/users/intro.adoc

```diff
@@ -1,2 +1,5 @@
-`bpmn-visualization` is a TypeScript library for visualizing process execution data on https://www.omg.org/spec/BPMN/2.0.2/[BPMN]
-diagrams. It is in early development stages and is subject to change prior to the `1.0.0` release.
+`bpmn-visualization` is a TypeScript library for visualizing processes and their execution data, in https://www.omg.org/spec/BPMN/2.0.2/[BPMN] format, with simplicity.
```

Comment:
ℹ️ I suggest we finalize the message in the README, then we will able to apply it in the user documentation